.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "C_Cpp.errorSquiggles": "Enabled",
+    "cmake.configureSettings": {
+        "CMAKE_TOOLCHAIN_FILE": "C:/dev/vcpkg/scripts/buildsystems/vcpkg.cmake"
```

Comment:
We should use `${vcpkgRoot}` in case the installation path is different. (Same as in `c_cpp_properties.json`)

Reply:
Hi @vi3itor,  `${vcpkgRoot}` is return `C:/dev/vcpkg/installed` instead of `C:/dev/vcpkg`, do you have any idea?
![image](https://user-images.githubusercontent.com/20337876/129718106-66bbaf6e-9ffa-48e3-bd83-ae66d1ec25f9.png)

Reply:
Hi @WesleyKhoiVo, it looks like the variable is [only works with the C++ extension](https://github.com/microsoft/vscode-cpptools/issues/4730). We probably should ask the user to set `VCPKG_ROOT` variable once and use it everywhere (similar to https://github.com/vi3itor/football/blame/windows/gfootball/doc/compile_engine.md#L46)